include/ddc/kernels/splines/spline_builder.hpp

```diff
@@ -10,10 +10,29 @@
 #include "deriv.hpp"
 
 namespace ddc {
+
+/**
+ * @brief An enum determining the backend solver of a SplineBuilder or SplineBuilder2d.
+ *
+ * An enum determining the backend solver of a SplineBuilder or SplineBuilder2d. Only GINKGO is available at the moment,
+ * other solvers will be implemented in the future.
+ */
 enum class SplineSolver {
-    GINKGO
-}; // Only GINKGO available atm, other solvers will be implemented in the futur
+    GINKGO ///< Enum member to identify the Ginkgo-based solver (iterative method)
+};
 
+/**
+ * @brief A helper giving the uniform/non_uniform status of a spline interpolation mesh according to its attributes.
+ *
+ * A helper giving the uniform/non_uniform status of a spline interpolation mesh according to its attributes.
```

Comment:
```suggestion
 * @brief A helper giving the uniform/non_uniform status of a spline interpolation mesh according to its attributes.
 *
 * A helper giving the uniform/non_uniform status of a spline interpolation mesh based on Greville abscissae according to its attributes.
```
As discussed this function gives the uniform/non-uniform status if Greville points are used.